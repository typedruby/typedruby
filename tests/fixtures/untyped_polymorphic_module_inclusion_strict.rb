# @typedruby

class Foo
  include Enumerable
end
