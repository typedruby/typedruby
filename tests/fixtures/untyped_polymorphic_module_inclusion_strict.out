info: Typechecking...
error: Too few type parameters supplied in instantiation of generic type

        @ __ROOT__/tests/fixtures/untyped_polymorphic_module_inclusion_strict.rb:4
      4 |    include Enumerable
                     ^^^^^^^^^^ here
